GothicVRProject/GothicVR
# Bugfix/music change

# To test

World:
1. Load world
2. Move away from start zone --> Music should change
3. Move into OC --> music should change

Lab:
1. On the left there is a dropdown. Select the first dropdown and hit Play
2. Music should be played
